src/target/stm32f1.c

```diff
@@ -163,7 +163,7 @@ bool gd32f1_probe(target *t)
 static bool at32f40_detect(target *t, const uint16_t part_id)
 {
 	// Current driver supports only *default* memory layout (256 KB Flash / 96 KB SRAM)
-	// (*) Support for external Flash for 512KB and 1024KB parts requires specific flash code (not implement)
+	// XXX: Support for external Flash for 512KB and 1024KB parts requires specific flash code (not implement)
```

Comment:
Should it say "not implemented", or "do not implement"? It is not clear.

Reply:
"not implemented" we believe, we'll update the comment

Reply:
thanks! :)